### PR TITLE
Clean DB package description, vingnettes, and examples

### PR DESCRIPTION
**Pull request by tonygardella**
## Description
In order to make pkgdown compatible with our packages. I'm going through and cleaning things that don't work with pkgdown. For the DB package I've:
1. Gone through each function to see what authors are there and added them to the description file.

2. Followed the Authors@R syntax:
```
Authors@R: person("Firstname", "Lastname", role = c("aut", "cre"), 
                                email = "address@gmail.com",
                                comment = c(ORCID = "0000-0000-0000-0000"))
```
3. Added \dontrun{} to examples in function headers where missing. They weren't breaking the build because I've set running examples to FALSE, but I'd like to make sure we make a habit of adding \dontrun{}

4. Set vignette R code to `eval =FALSE` because the code does not execute correctly when pkgdown automatically builds the vignettes.

@dlebauer @mdietze @infotroph @robkooper @bcow @ashiklom @istfer @serbinsh @ankurdesai Take a look at the DESCRIPTION file. I've added your email address. Let me know if it is wrong or you would like to not be listed.

## Review Time Estimate

- [ ] Immediately
- [x] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by infotroph** [comment]
> @dlebauer @mdietze @infotroph @robkooper @bcow @ashiklom @istfer @serbinsh @ankurdesai Take a look at the DESCRIPTION file. I've added your email address.

Since I'm not a DB package author, I'm guessing you tagged in this sentence for PR review rather than to check my address?

File: base/db/DESCRIPTION [hunk 1]
```diff
@@ -3,16 +3,38 @@ Type: Package
...
+                     email = "kooper@illinois.edu",
+                     comment = c(ORCID = "0000-0002-5781-7287")),
+             person("Shawn", "Serbin", role = c("aut", "cre"),
+                     email = "sserbin@bnl.gov",
+                     comment = c(ORCID = "0000-0003-4136-8971")),
+             person("Betsy", "Cowdery", role = c("aut", "cre"),
+                     email = "ecowdery@bu.edu",
+                     comment = c(ORCID = "0000-0002-6538-6296")),
+             person("Ankur", "Desai", role = c("aut", "cre"),
+                     email = "desai@aos.wisc.edu"
```
Comment (infotroph) [inline base/db/DESCRIPTION]: needs trailing comma

Comment (infotroph) [reply base/db/DESCRIPTION]: ...as do several more below

**Comment by mdietze** [comment]
I'm going to second @infotroph point about why we're adding `\dontrun` to examples that are intended to be run and serve as useful tests/checks.

**Comment by tonygardella** [comment]
@infotroph Thanks for the review! I did have you on the list of authors and forgot to add you. You helped write clone_pft. 

@infotroph @mdietze I jumped the gun on  adding don't run in the examples. I first and `pkgdown` run the examples and got a lot of errors so I was going to turn them all off and have the `pkgdown` example arg set to FALSE also until I could test them one at a time. But testing them isn't that hard so as I go through packages I'll make sure to flag ones that don't work and ones that do.

File: base/db/DESCRIPTION [hunk 2]
```diff
@@ -3,16 +3,41 @@ Type: Package
...
+                     email = "istfer@bu.edu",
+                     comment = c(ORCID = "0000-0001-8236-303X")),
+             person("Alexey", "Shiklomanov", role = c("aut", "cre"),
+                     email = "alexey.shiklomanov@pnnl.gov",
+                     comment = c(ORCID = "0000-0003-4022-5979")),
+             person("Tony", "Gardella", role = c("aut", "cre"),
+                     email = "tonygard@bu.edu",
+                     comment = c(ORCID = "0000-0003-4380-3412")),
+             person("Christopher", "Black", role = c("aut", "cre"),
+                     email = " ckb23@psu.edu",
```
Comment (infotroph) [inline base/db/DESCRIPTION]: Remove space inside quotes, otherwise 👍

File: base/db/DESCRIPTION [hunk 3]
```diff
@@ -3,16 +3,41 @@ Type: Package
...
-             person("David","LeBauer"),
-             person("Xiaohui", "Feng"),
-             person("Dan"," Wang"),
-             person("Carl", "Davidson"),
-             person("Rob","Kooper"),
-             person("Shawn", "Serbin"))
-Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang, Carl
-    Davidson, Rob Kooper, Shawn Serbin
-Maintainer: David LeBauer <dlebauer@illinois.edu>
+Authors@R: c(person("Mike", "Dietze", role = c("aut", "cre"),
```
Comment (infotroph) [inline base/db/DESCRIPTION]: Check me on this, but my reading of `?person` is that only one "cre" role is allowed per package. I'm not sure why they called it "creator" and not "maintainer", but think "person who creates the releases / CRAN submissions" more than "person who wrote the first line of code"

Comment (tonygardella) [reply base/db/DESCRIPTION]: Yeah you were right.

Comment (infotroph) [reply base/db/DESCRIPTION]: And just to confirm: The change in maintainer from @dlebauer to @mdietze is intentional? If so, this looks ready to merge.

Comment (mdietze) [reply base/db/DESCRIPTION]: Not intentional on my side! I'd prefer that @dlebauer remain in charge of the DB package

**Pull request merged**